src/Altinn.Platform/Altinn.Platform.PDF/Dockerfile

```diff
@@ -1,11 +1,14 @@
-FROM maven:3-jdk-12
+FROM maven:3-jdk-13 AS build
 WORKDIR build
 COPY . .
 RUN mvn -Pprod package
 
+FROM openjdk:13-jdk-alpine AS final
+WORKDIR app
+COPY --from=build /build/target .
 # setup the user and group
 # the user will have no password, using shell /bin/false and using the group dotnet
-RUN groupadd -g 3000 dotnet && adduser -u 1000 -g dotnet -s /bin/false dotnet
+#RUN groupadd -g 3000 dotnet && adduser -u 1000 -g dotnet -s /bin/false dotnet
```

Comment:
Lets have a chat about this tomorrow @alt-how

Reply:
Since Alpine is a different distro there are some diffs in the naming of some commands.
Have a look at [this](https://github.com/Altinn/altinn-studio/blob/master/src/Altinn.Platform/Altinn.Platform.Profile/Profile/Dockerfile#L16) which is used in all our other alpine-based docker files.

Reply:
Already fixed. ;)